docs/LSPSupport.md

```diff
@@ -0,0 +1,196 @@
+# LSP support
+
+As `LSP4IJ` doesn't implement full specification of LSP, this section list the LSP implementation for IntelliJ

```

Comment:
`LSP4IJ` currently doesn't fully implement the LSP specification. This section lists the supported LSP features for IntelliJ: